packages/subproviders/README.md

```diff
@@ -18,60 +20,6 @@ If your project is in [TypeScript](https://www.typescriptlang.org/), add the fol
 ]
 ```
 
-## Usage
```

Comment:
Please don't merge it before we migrate this section somewhere else. Don't want to just loose it.